### PR TITLE
Player anims

### PR DESCRIPTION
**Pull request by nshooter3**

### CODE REVIEW

File: Assets/Plugins/FMOD/Resources/FMODStudioSettings.asset [hunk 1]
```diff
@@ -46,7 +46,7 @@ MonoBehaviour:
     Value: 0
   OverlaySettings:
   - Platform: 1
-    Value: 1
```
Comment (Ominous) [inline Assets/Plugins/FMOD/Resources/FMODStudioSettings.asset:49]: is this file supposed to be committed ?

**Review by Ominous — CHANGES_REQUESTED** [review]
Double check you want that FMOD file changed

File: Assets/Plugins/FMOD/Resources/FMODStudioSettings.asset (hunk 1 above)
Comment (mitjmcc) [reply Assets/Plugins/FMOD/Resources/FMODStudioSettings.asset:49]: Yes Nick said he intentionally changed that in voice chat

Comment (nshooter3) [reply Assets/Plugins/FMOD/Resources/FMODStudioSettings.asset:49]: Yes that change was intentional. I disabled the fmod debug window.

**Review by Ominous — APPROVED** [review]

**Pull request merged**